convert2rhel/subscription.py

```diff
@@ -390,3 +390,44 @@ def rollback():
         unregister_system()
     except OSError:
         loggerinst.warn("subscription-manager not installed, skipping")
+
+
+def download_subscription_manager_packages():
+    from convert2rhel.systeminfo import system_info
+    from convert2rhel.utils import store_content_to_file
+    from convert2rhel.pkghandler import call_yum_cmd
+
+    _CENTOS_CONTRIB_REPO = '[contrib]\n' \
+                           'name=CentOS-6.10 - Contrib\n' \
+                           '#mirrorlist=http://mirrorlist.centos.org/?release=$releasever&arch=$basearch&repo=contrib\n' \
+                           'baseurl=http://mirror.centos.org/centos/6.10/contrib/$basearch/\n' \
+                           'gpgcheck=1\n' \
+                           'enabled=1\n' \
+                           'gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-6'
+
+    _TEMP_REPOFILE_DIR = '/tmp/convert2rhel/'
+    _TEMP_REPOFILE = _TEMP_REPOFILE_DIR + 'centos_contrib.repo'
+
+    version_full_number = re.search(r"([\d.]+)", system_info.system_release_file_content).group(1)
+    version_major = re.search(r"^(\d+)?\.?(\d+)?\.?(\*|\d+)?$", version_full_number).group(1)
+    version_minor = re.search(r"^(\d+)?\.?(\d+)?\.?(\*|\d+)?$", version_full_number).group(2)
+
+    if int(version_major) <= 6 and int(version_minor) <= 6:
+        logging.error(msg='Unable to download subscription-manager on CentOS/Oracle Linux 6.6 and older. '
+                          'Please use the --disable-submgr option or update to CentOS/Oracle Linux 6.7 or later')
+        sys.exit(1)
+
+    if int(version_major) == 6:
+        if not os.path.isdir(_TEMP_REPOFILE_DIR):
+            os.mkdir(_TEMP_REPOFILE_DIR)
```

Comment:
What if there is no left space on the disk.
Maybe, add some `logging.error`

Reply:
deal with no left space it's soo hard, that's the reason we don't deal with that on the tool, we let yum take care of it..